roles/thelounge/vars/main.yaml

```diff
@@ -0,0 +1,3 @@
+_thelounge__host: '{{ "127.0.0.1" if thelounge__apache_proxy | d(False) else
+  thelounge__host }}'
```

Comment:
```suggestion
_thelounge__host: >-
  {{ '127.0.0.1' if _thelounge_reverse_proxy else  thelounge__host }}
```

Reply:
Thanks, I removed it because I removed the apache proxy.